app/webpack/reactjs/components/admin/tests/hello_react.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+interface WelcomeProps {
+  message: string
+}
+
+export const Welcome: React.FC<WelcomeProps> = ({ message }) => (
```

Comment:
nit:  shouldn't this be called `welcome.tsx`?

Reply:
Yea this is super drafty for now. After we decide this is the solution to go, I'll clean up everything.